.github/workflows/benchmarks.yml

```diff
@@ -7,125 +7,85 @@ permissions:
   contents: read
 
 jobs:
-  benchmarks-ubuntu:
-    name: Ubuntu
+  benchmarks:
+    name: Benchmarks
+    env:
+      BUILD_DIR : "${{github.workspace}}/build/"
+      HWLOC_PATH: "${{github.workspace}}/build/vcpkg/packages/hwloc_x64-windows"
     strategy:
       matrix:
-        os: ['ubuntu-22.04']
-        build_type: [Release]
-        compiler: [{c: gcc, cxx: g++}]
-        shared_library: ['ON', 'OFF']
+        include:
+          - os: ubuntu-latest # shared library on Linux
+            shared_library: 'ON'
+            pool_scalable: 'ON'
+            pool_jemalloc: 'ON'
+
+          - os: ubuntu-latest # static library on Linux
+            shared_library: 'OFF'
```

Comment:
do we have any perf difference between shared lib ON and OFF? If not, I would test only shared lib and simplify this matrix

Reply:
Done